packages/amplify_datastore/android/src/main/kotlin/com/amazonaws/amplify/amplify_datastore/AmplifyDataStorePlugin.kt

```diff
@@ -144,6 +145,38 @@ class AmplifyDataStorePlugin : FlutterPlugin, MethodCallHandler {
                 }
         )
     }
+
+    @VisibleForTesting
+    fun onDelete(flutterResult: Result, request: HashMap<String, Any>) {
+        try {
+            var modelName = request["modelName"] as String
+            var modelData = request["model"] as HashMap<String, Any>
```

Comment:
Any request validation we want to do here?

Reply:
I don't think so - this is a required parameter in the dart interface.

Reply:
You can actually pass null to a required param. Also the interface might be incorrectly updated in the future. 
I think we did validation for these kinds of params in the other plugins as well.